dictcc.el

```diff
@@ -271,11 +282,23 @@ At the moment they are of the form `<tr id='trXXX'></tr>'."
                                (dictcc--insert-candidate #'cdr))))))
     (helm :prompt "Filter: " :sources (list source))))
 
+(defun dictcc--select-sourcelang-helm ()
+  "Select source language with helm."
+  (let* ((source `((name . ,"Languages" )
+		   (candidates . ,dictcc-source-languages-alist)
+		   (action . ,(quote identity))
+		   )))
+  (helm :prompt "Filter: " :sources (list source))))
```

Comment:
`dictcc.el` works with helm as well as ivy and therefore this functionality should to. But here it should be really easy to support both with `completing-read` which selects the users preferred backend automatically.